pages/1.13/security/oss/login/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+navigationTitle: Login
+title: Login
+excerpt: Logging in to your DC/OS cluster
+menuWeight: 20
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# User login
+
+With DC/OS, login is the process of exchanging user credentials for a [DC/OS Authentication token](/1.13/security/oss/authentication/authentication-token/).
+
+Obtaining a DC/OS Authentication is required by any user before being able to use a DC/OS cluster. In DC/OS the life-time of an Authentication token is limited to five days. Once the Authentication token expires the user must log in again.
+
+DC/OS handles multiple user types. User accounts can be managed via the [IAM API](/1.13/security/oss/iam-api/); see [User Management](/1.13/security/oss/user-management/).
+
+Different login methods exist for different user types but each one yields a DC/OS authentication token:
```

Comment:
1. You can end line #20 with a period. 
2. Add the below sentence in line #21:
The following are three types of login methods which exists for different user types: